lib/money/currency.rb

```diff
@@ -24,7 +26,18 @@ def find(currency_iso)
       end
 
       def currencies
-        @@currencies ||= Loader.load_currencies
+        @@currencies_mutex.synchronize do
+          @@currencies ||= Loader.load_currencies
+        end
+      end
+
+      def clear_cache
```

Comment:
let's get rid of this method, we don't want to reset this list unless the app is restarted